### PR TITLE
Reduce toggles

### LINKED ISSUE #63: Enhancement - Reduce Toggles
We've decided that it's best for user experience to collapse both photo toggles and name toggles under a single toggle per social network - as @1hauck says: `it’s like saying you only want to be half racist`

This means that for each network there should only be one toggle to enable obfuscation across names and photos.

Acceptance for this ticket does _not_ require any alteration to the obfuscation logic - we can simply have the single toggle call both toggle actions for now.

### PR DESCRIPTION
**Pull request by hyperwidget**
This closes #63 

This is a _HUGE_ change (kinda, but not really - just a change that had to be replicated a bunch of times (which makes me think of a next step refactor!).

The TLDR is: 
* Reduces the amount of toggles to one per app (no longer separating out names/photos)
  * This meant that we could remove of a lot of code that existed purely to support that
* Removes all traces of twitter and facebook
* Cleaned up some selectors to match the current state of targets
* Minor style updates in the toggler panel

updated

image

### CODE REVIEW

File: src/contentScripts/networks/angellist/handler.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import { createModel, getInitialVal } from '../../utils'
 import { URLS } from '../../utils/urls'
 
-export const TOGGLE_ANGELLIST_PHOTOS = 'toggleAlPhotos'
-export const TOGGLE_ANGELLIST_NAMES = 'toggleAlNames'
+export const TOGGLE_ANGELLIST_PHOTOS = 'toggleAngelListPhotos'
```
Comment (bhavyata9) [inline src/contentScripts/networks/angellist/handler.js:4]: Do we still need 2 constants? I see that they were removed in other places like LinkedIn?

**Review by bhavyata9 — APPROVED** [review]
LGTM, except for the minor comment for AngelList. The extension looks a lot cleaner now. Thanks for doing this!

File: src/contentScripts/networks/angellist/handler.js (hunk 1 above)
Comment (hyperwidget) [reply src/contentScripts/networks/angellist/handler.js]: We sure don't, thanks!

File: src/contentScripts/utils/urls.js [hunk 2]
```diff
@@ -1,11 +1,9 @@
 export const URLS = {
   linkedIn: 'linkedin.com',
-  twitter: 'twitter.com',
   angelList: 'angel.co',
   replit: 'repl.it',
   greenhouse: 'greenhouse.io',
-  lever: 'lever.co',
-  facebook: 'facebook.com',
+  lever: 'hire.lever.co',
```
Comment (dominiquesr) [inline src/contentScripts/utils/urls.js:6]: Could this be an array or regex? We're about to launch in the eu with `hire.eu.lever.co`

Comment (hyperwidget) [reply src/contentScripts/utils/urls.js:6]: Hi Dominique!

I don't see any reason why we wouldn't be able to leverage an array here with some work to our generic handlers; I'll add it into our todos!

**Pull request merged**